src/java/voldemort/VoldemortClientShell.java

```diff
@@ -151,21 +151,38 @@ public void process(boolean printCommands) {
     public static void main(String[] args) throws Exception {
 
         OptionParser parser = new OptionParser();
-        parser.accepts("client-zone-id", "client zone id for zone routing")
+        parser.accepts("client-zone-id", "Client zone id for zone routing")
               .withRequiredArg()
               .describedAs("zone-id")
               .ofType(Integer.class);
+        parser.accepts("config-file", "A properties file that contains client config properties")
+              .withRequiredArg()
```

Comment:
No, this means that if you give --config-file as an option, it requires the argument <file> to it.